scripts/dfx-orbit.validate.backend.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ -z "${RELEASE_COMMIT}" ]; then
+  echo "RELEASE_COMMIT is unset or set to the empty string"
+  exit 1
+fi
+
+# Checkout
+git fetch
+git checkout $RELEASE_COMMIT
```

Comment:
is it worth to add a `git pull` after this one? just to be sure?

plus, adding some check so that if the checkout/pull fails, it exits?

Reply:
Pull is effectively a fetch + checkout local branch to latest version. So it doesn't really make sense to 'pull' before doing a checkout.

Wrt to exiting, I'll add it in

Reply:
Added in both scripts